core/src/main/scala/chisel3/VerificationStatement.scala

```diff
@@ -30,6 +31,9 @@ trait VerifPrintMacrosDoc
 
 object assert extends VerifPrintMacrosDoc {
```

Comment:
I am a little confused, does this `object assert` not conflict with the one in `scala-2/chisel3/VerificationStatementMacros.scala`?

Reply:
Oh I see, it's `chisel3.VerifStmtMacrosCompat.assert`